src/lib/components/Cytoscape.react.js

```diff
@@ -955,6 +1051,8 @@ Cytoscape.defaultProps = {
     responsive: false,
     clearOnUnhover: false,
     elements: [],
+    contextMenuData: [],
+    tapContextMenu: {}
```

Comment:
For a read-only prop like `tapContextMenu` I don't think there's any value in setting a default, in fact it can lead to confusion because if you listen to the initial value of this in a callback I think you'll get `None`, NOT the default `{}`.